src/SqlAsyncCollector.cs

```diff
@@ -220,12 +222,11 @@ private static IEnumerable<string> GetExtraProperties(IDictionary<string, string
         /// <param name="table">Information about the table we will be upserting into</param>
         /// <param name="rows">Rows to be upserted</param>
         /// <returns>T-SQL containing data for merge</returns>
-        private static void GenerateDataQueryForMerge(TableInformation table, IEnumerable<T> rows, out string newDataQuery, out string rowData)
+        private static void GenerateDataQueryForMerge(TableInformation table, IEnumerable<T> rows, StringComparer comparer, out string newDataQuery, out string rowData)
```

Comment:
The table info already has the comparer so we don't need to pass it in separately right?

Reply:
Ah yes I can use table.Comparer. Thanks!